tests/bsim/bluetooth/audio/src/public_broadcast_sink_test.c

```diff
@@ -0,0 +1,423 @@
+/*
+ * Copyright 2023 Nxp
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_BT_PBP)
+
+#include <zephyr/sys/byteorder.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/audio/cap.h>
+#include <zephyr/bluetooth/audio/bap_lc3_preset.h>
+#include <zephyr/bluetooth/audio/pacs.h>
+#include <zephyr/bluetooth/audio/pbp.h>
+
+#include "common.h"
+
+#define SEM_TIMEOUT K_SECONDS(5)
+#define PA_SYNC_SKIP         5
+#define SYNC_RETRY_COUNT     6 /* similar to retries for connections */
+
+extern enum bst_result_t bst_result;
+
+static bool pbs_found;
+
+static K_SEM_DEFINE(sem_pa_synced, 0U, 1U);
+static K_SEM_DEFINE(sem_base_received, 0U, 1U);
+static K_SEM_DEFINE(sem_syncable, 0U, 1U);
+static K_SEM_DEFINE(sem_pa_sync_lost, 0U, 1U);
+
+static struct bt_bap_broadcast_sink *broadcast_sink;
+static struct bt_le_per_adv_sync *bcast_pa_sync;
+
+static struct bt_bap_stream streams[CONFIG_BT_BAP_BROADCAST_SNK_STREAM_COUNT];
+static struct bt_bap_stream *streams_p[ARRAY_SIZE(streams)];
+
+static const struct bt_audio_codec_cap codec = BT_AUDIO_CODEC_CAP_LC3(
+	BT_AUDIO_CODEC_LC3_FREQ_48KHZ, BT_AUDIO_CODEC_LC3_DURATION_10,
+	BT_AUDIO_CODEC_LC3_CHAN_COUNT_SUPPORT(1), 40u, 60u, 1u, (BT_AUDIO_CONTEXT_TYPE_MEDIA));
```

Comment:
```suggestion
	BT_AUDIO_CODEC_LC3_CHAN_COUNT_SUPPORT(1), 40u, 155u, 1u, (BT_AUDIO_CONTEXT_TYPE_MEDIA));
```

You need to suppport at least 75 and up to 155 SDU sizes for 48KHz, and down to 40 for 16_2_1